src/renderer/App.vue

```diff
@@ -154,9 +129,6 @@ export default {
   }),
 
   computed: {
-    hasBlurFilter () {
```

Comment:
I'm getting the message on my devtools console: `Property or method "hasBlurFilter" is not defined on the instance but referenced during render.` Because of `hasBlurFilter` here: https://github.com/ArkEcosystem/desktop-wallet/blob/6eb4b85a3fa35b9e60b7a45f9e6f56d5ff87e1ab/src/renderer/App.vue#L83

Reply:
Looks like the `button-dropdown` got added in one of the merges, I'll need to take a look at how it uses the blur class.

Reply:
Ok, looks like the `blur` on the button dropdown can be safely removed. Strangely the button dropdown wasn't positioned correctly for me during development (`this.$refs.buttonDropdown` always being `undefined`) while it displays correctly on 2.8.0, therefore I added a resize handler to also reposition the dropdown while it is open. @alexbarnsley could you have a look at this also? In particular 85700dc which adds the resize handler.